handlers/gravity_23.0.yml

```diff
@@ -0,0 +1,18 @@
+---
+# handlers for gravity as used in galaxy version 23.0
+
+- name: galaxy gravity restart
+  command: "{{ galaxy_venv_dir }}/bin/galaxyctl -c {{ galaxy_config_file }} graceful"
+  environment:
+    GRAVITY_STATE_DIR: "{{ galaxy_gravity_state_dir }}"
+  listen: "restart galaxy"
+  become: yes
+  become_user: "{{ __galaxy_user_name }}"
```

Comment:
This would fail on Main since the playbook connects as a different unprivileged user that can't sudo to other users. That said, in my case I have disabled the role handlers by setting `galaxy_systemd_mode` to `null` and defined my own, so maybe that is sufficient.

Reply:
There are some different ways in this role to do that, is there one that would work ?